src/Service/PagePurger.php

```diff
@@ -43,6 +51,41 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.7
+	 *
+	 * @brief Purge multiple pages
+	 *
+	 * Purges all the pages of the Pages object
+	 * by submitting a 'purge' action to the mediawiki
+	 * api with the parameter 'pageids' set to be the
+	 * pages ids in multiple-value seperation.
+	 *
+	 * @param Pages $pages the pages that are going to be purged
+	 *
+	 * @return bool
```

Comment:
I think for a single purge the boolean return may still make more sense?
But I don't have a strong opinion either way, returning a Page / null may also make sense!